api.planx.uk/.eslintrc

```diff
@@ -16,7 +16,7 @@
         "argsIgnorePattern": "^_"
       }
     ],
-    "@typescript-eslint/no-non-null-assertion": "warn",
+    "@typescript-eslint/no-non-null-assertion": "off",
```

Comment:
I've got somewhat of a solution for this (it's generally used for env vars which we can handle a bit better). Inspired by this... https://www.youtube.com/watch?v=q1im-hMlKhM

Reply:
Another benefit would be that we can validate all env vars up front and lose the dispersed `assert()` statements across the API

Reply:
In general, I'm finding that the non-null assertion is an easy way to avoid more complicated casts and type checks. It's a bit ugly but, from my point of view, hits the sweet spot between being strictly typed and easy to follow.